spec/lib/email_alert_signup_api_spec.rb

```diff
@@ -11,18 +11,19 @@
   }
   let(:available_choices) {
     [
-      OpenStruct.new(
+      {
+
```

Comment:
Looks like there's an errant line here

Reply:
Oops!  I've fixed this in a rebase.  Thanks for the eagle-eyed check!